File: #0215.kth-largest-element-in-an-array.cpp
class Solution {
public:
    int findKthLargest(vector<int>& nums, int k) {

        priority_queue<int, vector<int>, greater<int>> result;
        for (int num : nums) {
            result.push(num);
            if (result.size() > k) {
                result.pop();
            }
        }
        return result.top();
    }
};

File: #0215.kth-largest-element-in-an-array.js
/**
 * @param {number[]} nums
 * @param {number} k
 * @return {number}
 */
var findKthLargest = function(nums, k) {
    let arr = nums.sort((a, b) => (a - b));
    return arr[arr.length - k];
};
